packages/vm/benchmarks/mainnetBlocks.ts

```diff
@@ -0,0 +1,92 @@
+import * as fs from 'fs'
+import { bufferToInt } from 'ethereumjs-util'
+import blockFromRPC from '@ethereumjs/block/dist/from-rpc'
+import { Block } from '@ethereumjs/block'
+import VM from '../dist'
+import { getPreState, getBlockchain } from './util'
+import Common from '@ethereumjs/common'
+
+const BLOCK_FIXTURE = 'benchmarks/fixture/blocks-prestate.json'
+
+const onAdd = async (vm: VM, block: Block, receipts: any) => {
+  const vmCopy = vm.copy()
+
+  let result = await vmCopy.runBlock({
+    block,
+    generate: true,
+    skipBlockValidation: true,
+  })
+
+  // verify the receipt root, the logs bloom and the gas used after block execution, throw if any of these is not the expected value
+  if (
+    result.receiptRoot &&
+    result.receiptRoot.toString('hex') !== block.header.receiptTrie.toString('hex')
```

Comment:
small suggestion: compare buffers directly
```suggestion
    !result.receiptRoot.equals(block.header.receiptTrie)
```